docs/release-notes/4646-default-average_training_metrics-true.txt

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Breaking Changes**
+
+-  Experiment: the ``average_training_metrics`` experiment configuration default has been changed to
+   ``True``.
```

Comment:
Please elaborate a bit more here.  If I'm the user, what action do I need to take as a result of this "breakage"?  Or how can I decide if I need to take action at all?